galaxykubeman/Chart.yaml

```diff
@@ -24,3 +24,6 @@ dependencies:
     alias: rabbitmq
     tags:
       - deploy-rabbitmq
+  - name: galaxy
```

Comment:
The Galaxy chart is installed via a job: https://github.com/galaxyproject/galaxykubeman-helm/blob/2308a84ceaa1e8456f0d0bcfb6bed4936918166a/galaxykubeman/templates/job-setup-galaxy.yaml#L26
IIRC, this was so that the uninstallation could be done cleanly. Otherwise, the provisioners get uninstalled before the rest of the chart and leaves stuck resources.

Reply:
That commit (#0f8bd09) should have been reverted/rebased before my push... I'll clean that up.